app/javascript/components/widget-controller.jsx

```diff
@@ -160,12 +163,16 @@ export class WidgetController extends React.Component {
 
     const fetchPolicy = currentWidgetId === 0 ? 'network-only' : 'cache-first';
 
+    const cityName = this.props.location.pathname.substring(1);
+
     return (
       <Query
         query={getWidgets}
-        variables={{ id: currentWidgetId }}
+        variables={{ id: currentWidgetId, loc: cityName }}
```

Comment:
Would it be clearer for us to match `cityName` or `location` instead of `loc` in our graphql API?

Reply:
Will go for cityName